src/serverEntry.tsx

```diff
@@ -43,6 +46,22 @@ const server = createKoaServer({
   publicPath: '/new-member-assets',
   assetLocation: __dirname + '/assets',
 })
+
+server.router.use(
+  bodyParser({
+    extendTypes: { json: ['application/csp-report'] },
+  }),
+)
+
+if (process.env.USE_HELMET === 'true') {
+  appLogger.info('Using helmet and strict CSP ⛑')
+  server.app.use(helmet)
+  server.router.use(helmet)
+} else if (process.env.NODE_ENV !== 'development') {
+  appLogger.warn(
```

Comment:
I would even consider bailing at this point. Going without in production would not be great as the log statement says!

Reply:
I don't agree. I.e. if we're putting up a PR instance or want to try something locally it's not obvious we care about CSPs.